content/01.proposal.md

```diff
@@ -0,0 +1,54 @@
+## Proposal {.page_break_before}
+
+Manubot is a system for writing scholarly manuscripts on GitHub.
+Manubot aims to **transform publishing** to be transparent & reproducible, immediate & permissionless, versioned & automated, collaborative & open, linked & provenanced, decentralized & hackable, interactive & annotated, and free of charge.
```

Comment:
This list is pretty extensive. I think you should make clear that this is a long term goal:
"Our __long term__ goal with manubot is to transform publishing to be..." 
also `free of charge` doesn't specify the type of freedom. Free to read, improve, and mine?

Reply:
These are not just long-term goals... they are what the Manubot provides today to anyone who decides to use it!

Reply:
> also "free of charge" doesn't specify the type of freedom. Free to read, improve, and mine?

I just meant that you don't have to pay to use Manubot... anyone can produce publication quality manuscripts without buying any products (Word, EndNote, a journal, etcetera).